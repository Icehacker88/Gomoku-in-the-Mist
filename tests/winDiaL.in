place A1
place A19
place B1
place B18
place B2
place C17
place B3
place D16
place B4
place E15
