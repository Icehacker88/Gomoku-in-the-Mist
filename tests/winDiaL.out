White wins!
A1A19B1B18B2C17B3D16B4E15
Thank you for playing!
